src/routing_table/mod.rs

```diff
@@ -300,9 +301,10 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
 
     pub fn is_in_our_group(&self, name: &T) -> bool {
         if self.our_group_prefix.matches(name) {
-            return unwrap!(self.groups.get(&self.our_group_prefix)).contains(name);
+            unwrap!(self.groups.get(&self.our_group_prefix)).contains(name)
```

Comment:
I'm not sure I see the point in this - we have no official policy, but I normally favour an early `return` over an `else` block.  Should we maybe establish a guideline so we don't end up playing ping-pong over lines like these?